### PR TITLE
Update tr-7.md : Availability of foundational model

### PR DESCRIPTION
**Pull request by alvin-c-shih**
THE FOLLOWING DISCLAIMER APPLIES TO ALL SOFTWARE CODE AND OTHER MATERIALS CONTRIBUTED IN CONNECTION WITH THIS SOFTWARE:
THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS “AS IS” AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.
ONLY THE SOFTWARE CODE AND OTHER MATERIALS CONTRIBUTED IN CONNECTION WITH THIS SOFTWARE, IF ANY, THAT ARE ATTACHED TO (OR OTHERWISE ACCOMPANY) THIS SUBMISSION (AND ORDINARY COURSE CONTRIBUTIONS OF FUTURES PATCHES THERETO) ARE TO BE CONSIDERED A CONTRIBUTION. NO OTHER SOFTWARE CODE OR MATERIALS ARE A CONTRIBUTION.

### CODE REVIEW

File: docs/_threats/tr-7.md [hunk 1]
```diff
@@ -1,14 +1,65 @@
...
+    - "Strengthening the Resilience of Outsourced Technology Services"
 ---
 
-Hosted LLMs typical have usage limits, but often lack the sophisticated controls needed to manage this.
+RAG systems are proliferating due to the low barrier of entry compared to traditional chatbot technologies. RAG has applicability for internal users as well as supporting customer-related use cases.
 
-Given the commercial model of LLM services is typically on a per usage basis, there is the risk that significant resources can be expended via Chat App if uncontrolled access is allowed. e.g. DOS, injecting very large contexts, brute forcing to elicit very long responses, etc.
+Many LLMs require GPU compute infrastructure to provide an acceptable level of responsiveness. Furthermore, some of the best-performing LLMs are proprietary. Therefore, the path of least resistance is to call out to a Technology Service Provider (TSP).
+
+However, applications that index sensitive information may use on-prem GPUs because:
```
Comment (ColinEberhardt) [inline docs/_threats/tr-7.md:19]: at the moment, on-prem hosting of the foundational model is out of scope for the use case we are exploring - see: 

https://github.com/finos/ai-readiness/blob/main/docs/index.md?plain=1#L25

File: docs/_threats/tr-7.md [hunk 2]
```diff
@@ -1,14 +1,65 @@
...
+    - Unexpectedly long prompts due to large chunk sizes or many chunks packed into the prompt.
+        - This could be exacerbated by systems that work with multimedia content.
+        - LLM attacks to exfiltrate training data may also generate the maximum number of tokens.
+    - Scripts which repeatedly hit API endpoints without built-in throttling.
+        - This may result from Strats searching for new sources of "signal".
+    - Agentic systems which may generate additional LLM / search calls that were not taken into account in the initial design / capacity planning. 
+-	**TSP Outages**:
+    - Some LLM providers might not have sufficient technical maturity to provide uptime to meet target SLA.
+    - Inability to fail over to an equivalent LLM back end due to proprietary model / lack of capacity at other TSPs. (FFIEC Appendix J).
+- **On-Prem Capacity**:
```
Comment (ColinEberhardt) [inline docs/_threats/tr-7.md:38]: as per above, this is out of scope

File: docs/_threats/tr-7.md [hunk 3]
```diff
@@ -1,14 +1,65 @@
...
+    - BCP site may not have equivalent GPU capacity.
+    - Load testing new LLM serving libraries and settings.
+- **VRAM Exhaustion**:
+    - VRAM utilization can increase for a variety of reasons:
+        - new release of LLM library has a memory leak
+        - new caching techniques to improve throughput at the cost of VRAM
+        - setting a longer context length
+        - etc.
+
+# Recommendations:
```
Comment (ColinEberhardt) [inline docs/_threats/tr-7.md:49]: threats should describe the problem, with mitigations described via the associated controls. Can you consider whether your recommendations are covered by our existing controls, or whether additional ones are needed?

**Review by ColinEberhardt — CHANGES_REQUESTED** [review]
Thanks, really good contribution, but a few comments to address.

File: docs/_threats/tr-7.md (hunk 3 above)
Comment (alvin-c-shih) [reply docs/_threats/tr-7.md:49]: I'll take out that section.  I'll look into whether the first couple of points can go into CT-8.

**Review by ColinEberhardt — APPROVED** [review]
